crest/Assets/Crest/Crest/Scripts/LodData/RegisterClipSurfaceInput.cs

```diff
@@ -214,11 +257,11 @@ private void LateUpdate()
                 return;
             }
 
-            if (_currentMaterial != _material)
```

Comment:
i believe this was the only thing that used _material so I took the liberty to remove it and just get at the material directly in the below